kernel_gateway/tests/test_gatewayapp.py

```diff
@@ -81,7 +81,6 @@ def test_trust_xheaders(self):
 
     def test_ssl_options(self):
         app = KernelGatewayApp()
-        os.environ['KG_CERTFILE'] = '/test/fake.crt'
```

Comment:
So, in the test here, we have to have this certfile or key or ca_cert in order to check the default case i.e., if the `ssl_version` is set to `5` or not.

Reply:
Correct.  I would perform two scenarios in this test.  One that doesn't set the certfile (or keyfile or ca_cert) and asserts ssl_options is None, the other that does set certfile and asserts the default.

Reply:
Right :) Cool. I'll make that change :)